src/anim/controller/anim-controller.js

```diff
@@ -320,7 +320,9 @@ class AnimController {
         let state;
         let animation;
         let clip;
-        this.previousState = transition.from;
+        // If transition.from is set, tranisition from the active state irregardless of the transition.from value (this could be the previous, active or ANY states).
```

Comment:
```suggestion
        // If transition.from is set, transition from the active state irregardless of the transition.from value (this could be the previous, active or ANY states).
```